package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "smartevents-ui",
   "version": "0.0.1",
-  "federatedModuleName": "smartevents",
+  "federatedModuleName": "smart_events",
```

Comment:
I wonder whether it would be better to make this consistent with [this](https://github.com/RedHatInsights/cloud-services-config/pull/1067/files#diff-c608be8e36b8dbaba0b0fc0d75c28bc58ab3a27167774c46f31f14319931c693R786)?

Reply:
Unfortunately I cannot use dash here. Federated modules plugin doesn't allow it and throws an error. Switched then to snake case.

Reply:
I s the readme fine without updates?
```
npm install
npm run start:federate

This will run a dev server on http://localhost:9006 that will serve a federated module named `smartevents`.
```

Reply:
good catch. fixed.